src/components/v2/V2Project/V2PayButton/V2ConfirmPayModal/NftRewardCell.tsx

```diff
@@ -0,0 +1,63 @@
+import { Tooltip } from 'antd'
+import ExternalLink from 'components/ExternalLink'
+import Loading from 'components/Loading'
```

Comment:
**suggestion** I think we can prefer using the antd icon directly here: `LoadingOutlined`